### PR TITLE
(Feature) Manage Created Crowdsale Contracts (choose contract button + popup)

### PR DESCRIPTION
**Pull request by fernandomg**
This PR is the first step towards in the completion of #323 

- Choose crowdsale button for dev env
- Popup listing crowdsales

### CODE REVIEW

**Comment by vbaranov** [comment]
@fernandomg there are some conflicts after merging of https://github.com/oraclesorg/ico-wizard/pull/391. Could you, please, resolve it?

**Comment by fernandomg** [comment]
@vbaranov done!

**Comment by fernandomg** [comment]
@vbaranov thanks!
1. It's available only for development mode (https://github.com/oraclesorg/ico-wizard/pull/401/files#diff-805607bc24e7cead114b8868c9c4ab83R42)
2. Sure thing

File: src/App.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ class App extends Component {
           <Route exact path="/" component={crowdsaleAddr?Crowdsale:Home}/>
           <Route exact path="/crowdsale" component={Crowdsale}/>
           <Route exact path="/invest" component={Invest}/>
+          <Route exact path="/crowdsaleDetails" component={Contracts}/>
```
Comment (fvictorio) [inline src/App.js]: Can we use another path so that we don't confuse it with `/crowdsale`? Maybe `manageCrowdsale` or just `manage`?

Comment (fernandomg) [reply src/App.js]: I'll change it to `manage` as it seems the most accurate for the current routes naming

**Pull request merged**